.circleci/ci/accessibility-axe.js

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+const fs = require('fs');
+const ci = new (require('./ci.js'))();
+const AxeBuilder = require('@axe-core/webdriverjs');
+const WebDriver = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const { createHtmlReport } = require('axe-html-reporter');
+
+
+const calculateAccessibility = async () => {
```

Comment:
Can you format all the JS files, it is not readable right now ?

Reply:
done